crates/host/src/wasmbus/mod.rs

```diff
@@ -1438,24 +1438,12 @@ impl Host {
     }
 
     #[instrument(level = "debug", skip_all)]
-    async fn stop_actor(&self, actor: &Actor, host_id: &str) -> anyhow::Result<()> {
+    async fn stop_actor(&self, actor: &Actor, _host_id: &str) -> anyhow::Result<()> {
         trace!(component_id = %actor.id, "stopping component");
 
+        // TODO: How to know if abort was successful?
```

Comment:
This abort happens immediately and doesn't return a Result (and can't panic) so there's no additional feedback to gain here 😄 

If you mean in terms of knowing that a component finishes handling its current invocations, that might be worth an additional issue since that's something we could address

Reply:
I would prefer we remove this comment though unless it's tied to an active issue

Reply:
Removed the line